math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/OrdinaryLeastSquares.scala

```diff
@@ -0,0 +1,96 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements. See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership. The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License. You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the License is distributed on an
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  * KIND, either express or implied. See the License for the
+  * specific language governing permissions and limitations
+  * under the License.
+  */
+
+package org.apache.mahout.math.algorithms.regression
+
+import org.apache.mahout.math.{Vector => MahoutVector}
+import org.apache.mahout.math.drm.RLikeDrmOps._
+import org.apache.mahout.math.drm.DrmLike
+import org.apache.mahout.math.scalabindings._
+import org.apache.mahout.math.scalabindings.RLikeOps._
+
+import scala.reflect.ClassTag
+
+/**
+  * import org.apache.mahout.math.algorithms.regression.OrdinaryLeastSquares
+  * val model = new OrdinaryLeastSquares()
+  *
+  * model.calcStandardErrors = true
+  * 
+  */
+class OrdinaryLeastSquares[K](hyperparameters: Map[String, Any] = Map("" -> None)) extends LinearRegressor[K] {
+  // https://en.wikipedia.org/wiki/Ordinary_least_squares
+
+  var calcStandardErrors: Boolean = hyperparameters.asInstanceOf[Map[String, Boolean]].getOrElse("calcStandardErrors", true)
+  var addIntercept: Boolean = hyperparameters.asInstanceOf[Map[String, Boolean]].getOrElse("addIntercept", true)
+
+  var summary = ""
+  def fit(drmFeatures: DrmLike[K], drmTarget: DrmLike[K]): Unit = {
```

Comment:
Continuing on from our discussion on Slack I would think that Fit may be a more appropriate place for Hyperparameters eg:
```
fit(observed_independent: Drm[K], observerd_targets: Drm[K],  hyperparamters: Option[List[double]]): List[double]
```
I think that this may be a matter of Convention, so If you're following a convention that I am not familiar with, this may be fine.  However I feel that this may be more robust.